db/models.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+from db.database import Base
+
+
+class DBBook(Base):
+    __tablename__ = "book"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String(255), nullable=False)
+    summary = Column(String(511), nullable=False)
+    publication_date = Column(Date, nullable=False)
+    author_id = Column(Integer, ForeignKey("author.id"))
+
+
+class DBAuthor(Base):
+    __tablename__ = "cheese"
```

Comment:
The `__tablename__` for `DBAuthor` is incorrectly set to 'cheese'. It should be 'author' to match the table name used in the `ForeignKey` constraint in `DBBook`.